daemon/emer-daemon.c

```diff
@@ -1547,6 +1547,15 @@ emer_daemon_new (const gchar             *persistent_cache_directory,
                        NULL);
 }
 
+void
+emer_daemon_reload (EmerDaemon *self)
+{
+  EmerDaemonPrivate *priv = emer_daemon_get_instance_private (self);
+
+  if (priv->machine_id_provider)
```

Comment:
I prefer `if (priv->machine_id_provider != NULL)` for checks like this, and a quick skim of this file suggests that's the style in use here.